### PR TITLE
Avoid test case throw meaningless exception when sys env is not set.

### PR DESCRIPTION
**Pull request by katmsft**

### CODE REVIEW

**Review by ahsonkhan — APPROVED** [review]

**Review by RickWinter — APPROVED** [review]

File: sdk/storage/azure-storage-common/test/test_base.cpp [hunk 1]
```diff
@@ -31,14 +31,26 @@ namespace Azure { namespace Storage { namespace Test {
   constexpr static const char* c_AadClientId = "";
   constexpr static const char* c_AadClientSecret = "";
 
+  std::string GetEnv(const char* name)
```
Comment (Jinming-Hu) [inline sdk/storage/azure-storage-common/test/test_base.cpp:34]:
```suggestion
  std::string GetEnv(const std::string& name)
```

File: sdk/storage/azure-storage-common/test/test_base.cpp [hunk 2]
```diff
@@ -31,14 +31,26 @@ namespace Azure { namespace Storage { namespace Test {
...
+  {
+    const char* ret = std::getenv(name);
```
Comment (Jinming-Hu) [inline sdk/storage/azure-storage-common/test/test_base.cpp:36]:
```suggestion
    const char* ret = std::getenv(name.data());
```

File: sdk/storage/azure-storage-common/test/test_base.cpp [hunk 3]
```diff
@@ -31,14 +31,26 @@ namespace Azure { namespace Storage { namespace Test {
   constexpr static const char* c_AadClientId = "";
   constexpr static const char* c_AadClientSecret = "";
 
+  std::string GetEnv(const std::string& name)
+  {
+    const char* ret = std::getenv(name.data());
+    if (!ret)
+    {
+      throw std::runtime_error(
+          std::string(name)
```
Comment (Jinming-Hu) [inline sdk/storage/azure-storage-common/test/test_base.cpp:40]:
```suggestion
          name
```

Comment (Jinming-Hu) [reply sdk/storage/azure-storage-common/test/test_base.cpp:40]: reformat with clang-format

File: sdk/storage/azure-storage-common/test/test_base.cpp [hunk 4]
```diff
@@ -31,14 +31,26 @@ namespace Azure { namespace Storage { namespace Test {
   constexpr static const char* c_AadClientId = "";
   constexpr static const char* c_AadClientSecret = "";
 
+  std::string GetEnv(const std::string& name)
+  {
+    const char* ret = std::getenv(name.data());
+    if (!ret)
+    {
+      throw std::runtime_error(
+          name
```
Comment (Jinming-Hu) [inline sdk/storage/azure-storage-common/test/test_base.cpp:40]: reformat with clang-format

**Review by Jinming-Hu — APPROVED** [review]

**Pull request merged**